packages/terra-data-grid/src/WorklistDataGrid.jsx

```diff
@@ -165,14 +157,10 @@ function WorklistDataGrid(props) {
     onDisableSelectableRows,
     onEnableRowSelection,
     hasSelectableRows,
-    intl,
     rowHeaderIndex,
   } = props;
 
-  const rowSelectionEffectTriggered = useRef(false);
   const selectedRows = useRef([]);
```

Comment:
Shouldn't this variable have also been removed?  It was created to manage the row selection state for the aria live region logic.